content/posts/shortcodes/index.fr.md

```diff
@@ -1,73 +1,152 @@
 ---
-title: "Codes abrégés"
+
+title: "Les Shortcodes"
 date: 2023-11-05T14:06:25+02:00
 description: "Codes abrégés"
 menu:
   sidebar:
-    name: "Codes abrégés"
+    name: "Les Shortcodes"
     identifier: shortcodes
     weight: 700
 hero: boat.jpg
 ---
-Ce billet échantillon est destiné à tester les éléments suivants :
+
+Ce billet d'échantillon est destiné à tester les éléments suivants :
 
 - Manipulation d'une image.
 - Différents shortcodes.
 
 ## Alerte
 
-Les alertes suivantes sont disponibles dans ce thème.
+Ce thème propose différents types d'alertes pour votre publication. Par exemple, si on ajoute le code ci-dessous :
 
+```
 {{< alert type="success" >}}
```

Comment:
You have to use Hugo specific escape marker in shortcode so that they can be shown in rendered codeblocks. For example:
```
{{/*< alert type="success" */>}}
This is sample alert with `type="success"`.
{{</* /alert >*/}}
```

Notice the `/*` and `*/`.

Reply:
Hi ! OK, it will be better now. It's OK for You ?

Reply:
Not there yet. Added new comments. Can you please acknowledge them.